packages/calypso-products/src/translations.tsx

```diff
@@ -1102,7 +1102,14 @@ export const getJetpackProductsLightboxDescription = (): Record< string, Transla
 		'Seamlessly accept purchase orders as a payment method on your WooCommerce store.'
 	);
 	const woocommerceShippingLightboxDescription = translate(
-		'Print USPS and DHL labels right from the WooCommerce desktop and save up to 90% instantly. WooCommerce Shipping is free and saves you time and money.'
+		'Print USPS and DHL labels right from the WooCommerce desktop and save up to %(discountPercent)s instantly. WooCommerce Shipping is free and saves you time and money.',
+		{
+			args: {
+				discountPercent: numberFormat( 0.9, {
+					numberFormatOptions: { style: 'percent' },
+				} ),
+			},
+		}
```

Comment:
@rcoll can you help me test these changes? I couldn't quite figure it out from https://github.com/Automattic/wp-calypso/pull/90763 - I think it's just missing more detailed testing instructions 🙏

Reply:
thank you :-)